src/query/regu_var.hpp

```diff
@@ -266,4 +269,13 @@ REGU_VARIABLE_CLEAR_FLAG (regu_variable_node *regu, int flag)
   regu->flags &= ~flag;
 }
 
+DB_TYPE
+REGU_VARIABLE_GET_TYPE (const regu_variable_node *regu)
+{
+  if (regu && regu->domain && regu->domain->type)
+    {
+      return regu->domain->type->get_id();
+    }
+  return DB_TYPE_UNKNOWN;
+}
```

Comment:
```suggestion
DB_TYPE
REGU_VARIABLE_GET_TYPE (const regu_variable_node *regu)
{
  if (regu)
    {
      return TP_DOMAIN_TYPE (regu->domain);
    }
  return DB_TYPE_UNKNOWN;
}
```
`TP_DOMAIN_TYPE` macro is declared in object_domain.h. with this, we don't need to include "object_primitive.h" here. 
And maybe this inline function is not required.

Reply:
I fixed it.